vm/inc/ubpf.h

```diff
@@ -571,13 +579,18 @@ extern "C"
      * @param[in] registers Array of 11 registers representing the VM state.
      * @param[in] stack_start Pointer to the beginning of the stack.
      * @param[in] stack_length Size of the stack in bytes.
+     * @param[in] register_mask Bitmask of registers that have been modified since the start of the program.
+     * @param[in] stack_mask_start Bitmask of the stack that has been modified since the start of the program.
+     *  Each set bit represents 1 byte of the stack that has been modified.
```

Comment:
Is it important for the user to know whether the top of the stack is at the lower address in this bitmask or the higher address?